.gitmodules

```diff
@@ -1146,3 +1146,6 @@
 [submodule "vendor/grammars/zephir-sublime"]
 	path = vendor/grammars/zephir-sublime
 	url = https://github.com/phalcon/zephir-sublime
+[submodule "vendor/grammars/berry"]
+	path = vendor/grammars/berry
+	url = https://github.com/berry-lang/berry
```

Comment:
Linguist isn't going to be able to find and use the grammar in this repo. What's more, this repo is the entirety of the language from what I can see. We need a repo that contains _just_ the grammar.

Reply:
Does this dedicated repo need to be structured in any particular way before we go ahead and create it?

Reply:
It needs to be structured like other TextMate-compatible grammars.

This is where the grammar compiler expects the different grammar files to be:

https://github.com/github/linguist/blob/3d08e73be385c31cdf9f41371432dd37efb88d41/tools/grammars/compiler/loader.go#L119-L128

Reply:
We have relocated the TextMate grammar files into a dedicated repo as requested.  I have also removed the language tutorial samples and added two real world samples that are used in my own [heating controller](https://github.com/Beormund/Tasmota32-Multi-Zone-Heating-Controller) repo . I'm sorry I tried several times to get the LICENSE to register but it does not seem to have been cached even though it is present in the grammar repo.